dist/espree.d.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Tokenizes the given code.
+ * @param {string} code The code to tokenize.
+ * @param {ParserOptions} options Options defining how to tokenize.
+ * @returns {import('acorn').Token[] | null} An array of tokens.
+ * @throws {import('./lib/espree').EnhancedSyntaxError} If the input code is invalid.
+ * @private
+ */
+export function tokenize(code: string, options: ParserOptions): import('acorn').Token[] | null;
+/**
+ * Parses the given code.
+ * @param {string} code The code to tokenize.
+ * @param {ParserOptions} options Options defining how to tokenize.
+ * @returns {import('acorn').Node} The "Program" AST node.
+ * @throws {import('./lib/espree').EnhancedSyntaxError} If the input code is invalid.
+ */
+export function parse(code: string, options: ParserOptions): import('acorn').Node;
+export const version: "main";
+export const VisitorKeys: visitorKeys.VisitorKeys;
+export const Syntax: {
+    [x: string]: string;
+};
+export const latestEcmaVersion: number;
+export const supportedEcmaVersions: number[];
+export type ParserOptions = {
+    allowReserved?: boolean;
+    ecmaVersion?: import('acorn').ecmaVersion;
```

Comment:
I think we should have our own type for `ecmaVersion` because we may not support all the values that Acorn supports. For example, if Acorn adds `"next"`, that doesn't necessarily mean that Espree will allow `"next"`.